app/views/partials/foyer/capture-montant-ressource.html

```diff
@@ -30,7 +30,7 @@ <h3 class="panel-title">
       </div>
       <div class="col-sm-3">
         <label for="{{ ressource.type.id }}-year">
-          Total depuis {{ debutAnneeGlissante }}
+          Total de {{ debutAnneeGlissante }} à {{ months[months.length-1].label }}
```

Comment:
@monbocal Je préfère ne pas forcer le saut de ligne.
@MattiSG J'ai fait [Apply Find/Replace all 'MMMM YYYY' to 'MMMM YYYY' (non breakable space) ](https://github.com/sgmap/mes-aides-ui/pull/573/commits/097da361b305470f015ddcbb8fab06e00c3b52e5) (16 matches across 11 files). C'est un peu violent mais ça me parait pertinent (mais pas forcément dans cette PR...).